tox.ini

```diff
@@ -7,7 +7,7 @@ envlist =
 [gh-actions]
 python =
```

Comment:
Looks like this list is useless, when running tox on github actions with python versions that are not listed her still runs them:
- 3.12-dev: https://github.com/PyGithub/PyGithub/actions/runs/5259774567/jobs/9505791611#step:5:26
- 3.11: https://github.com/PyGithub/PyGithub/actions/runs/5259774567/jobs/9505791522#step:5:26

So what is the point in maintaining this list?

Reply:
oh， this make sure on ci job only run tests once

Reply:
right, I guess one entry is sufficient then

Reply:
Wait, I'll test if it will run multiple tests in one ci job.

Reply:
This is necessary.

comment added, also notice developer to keep python version in ci and tox.ini synced